src/js/components/Tag/__tests__/__snapshots__/Tag-test.tsx.snap

```diff
@@ -551,6 +551,7 @@ exports[`Tag onRemove 1`] = `
       </span>
     </div>
     <button
+      aria-label="Remove Test"
```

Comment:
We should probably have a test case for the other 2 variations

Reply:
will do